src/main/java/com/iota/iri/network/TCPNeighbor.java

```diff
@@ -77,12 +77,18 @@ public void setSink(Socket sink) {
         this.sink = sink;
     }
 
+    /**
+     * This is a non-blocking write and that makes it necessary to make a defensive copy of the sent data.
+     *
```

Comment:
remove newline (or add one in the UDP doc)

Reply:
done